src/utilities/fe_dofs.jl

```diff
@@ -0,0 +1,116 @@
+type FEField
```

Comment:
I think that would be possible to get around using two different `FEValues` for the two functions though, since that will be needed anyway, right? But it makes sense that a `FEField` has a function space over which it is defined.